### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by mateusz-brainhub**
Keywords NPM updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +80,19 @@
     "webpack-dev-middleware": "^1.11.0",
     "webpack-hot-middleware": "^2.20.0"
   },
-  "keywords": [],
+  "keywords": [
+    "react"
```
Comment (piotr-s-brainhub) [inline package.json]: @mateusz-brainhub 

here you missed a comma

Comment (mateusz-brainhub) [reply package.json]: done

Comment (mateusz-brainhub) [reply package.json]: Piotr could you have a look if now is OK?

Comment (piotr-s-brainhub) [reply package.json]: @mateusz-brainhub 

I asked a question why you had changed `dist` into `lib`

**Pull request merged**